### DISCUSSION TITLE
Disable Decimal.js Generation?

### DISCUSSION THREAD

**Discussion by tarazena** [prisma/prisma#5980] [Q&A] [answered]
Hey Guys

I have few columns in my db that have `Decimal` values, my GraphQL Server usually returns whatever responses that Prisma returns (using `Float` in GraphQL for Decimals).

Recently I upgraded to the latest version of Prisma (2.18.0, was on 2.15.0), Since Prisma generates Decimal for Decimal types of columns, All my GraphQL schema was broken (Now its returning Objects (Decimal.js) instead of numbers), and i was forced to revert back to my original version.

Is there a way to disable using Decimal.js?

**Comment by pantharshit00** [comment, marked as answer]
Hey @tarazena 

Please paste this into a new feature request so that we can use our standard feature request processing process for this :)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
